app/src/main/java/ch/epfl/sweng/SDP/game/RankingFragment.java

```diff
@@ -78,9 +108,39 @@ public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                     }
                 }
 
-                rankedUsernames = SortUtils.sortByValue(finalRanking);
-                ArrayAdapter<String> adapter = new RankingAdapter(getActivity(),
-                        rankedUsernames.toArray(new String[rankedUsernames.size()]));
+                String userNameId = account.getUsername();
```

Comment:
Are you trying to get the username or the userId? I think you want to get the ID here?

Reply:
I need the user name here since I compare them with the rankings in the database